image/golem_blender_app/golem_blender_app/commands/verify.py

```diff
@@ -32,12 +32,16 @@
     with zipfile.ZipFile(subtask_outputs_dir / f'{subtask_id}.zip', 'r') as f:
         f.extractall(subtask_results_dir)
 
-    subtask_num = utils.get_subtask_num_from_id(subtask_id)
-
     with utils.get_db_connection(work_dir) as db:
-        utils.update_subtask(db, subtask_num, utils.SubtaskStatus.VERIFYING)
+        subtask_num = utils.get_subtask_num(db, subtask_id)
+        utils.update_subtask_status(
+            db,
+            subtask_id,
+            utils.SubtaskStatus.VERIFYING)
+        dir_contents = os.listdir(subtask_results_dir)
```

Comment:
Maybe we can use `Pathlib` over `os.`?
```suggestion
        dir_contents = subtask_results_dir.iterdir()
```

Reply:
Will do. But I cannot commit this suggestion since `subtask_results_dir.iterdir()` already includes the parent directory(ies) in each entry, I need to modify line 44 as well